### DISCUSSION TITLE
Corne 5 colums layout

### DISCUSSION THREAD

**Discussion by tali-soupBrain** [nickcoutsos/keymap-editor#243] [Q&A]
Hi!

I was wondering if it is possible to use this application for a Corne keyboard 5-columns?
(Sorry if I missed the option to do so)

**Comment by nickcoutsos** [comment]
Yes, make sure your keymap file includes the proper matrix transform, e.g.

```dts
/ {
  chosen {
    zmk,matrix-transform = &five_column_transform;
  };
};
```

If it can detect that it'll switch to the appropriate layout definition.

**Reply by tali-soupBrain** [reply]
Works perfectly thanks!!
